### DISCUSSION TITLE
google nest hub --> implementazione con HASS.io

### DISCUSSION THREAD

**Discussion by RobyBS** [indomus/forum#575] [HUB: Home Assistant]
Ciao a tutti ... 
finalmente ho deciso di fare quanto in oggetto ... 

ho seguito la guida , che è dettagliatissima e precissisima ... 

ma al momento del test sulla console action di google, mi appare il seguente messaggio ...

warning
We're sorry, but something went wrong. Please try again.
"abra cadabra" ha abbandonato la conversazione

e quando dal cell avvio la action mi dice , dopo aver inserito le credenziali di HA :
could not reach test , try later ...

dove sbaglio ? 

nel configuration.yaml ho inserito anche : 

```
# Example configuration.yaml entry
sensor:
  - platform: google_wifi
 
```
e nel log di Hassio vedo questo errore : 

Logger: homeassistant.components.google_wifi.sensor
Source: components/google_wifi/sensor.py:152
Integration: google_wifi (documentation, issues)
First occurred: 10:41:28 (113 occurrences)
Last logged: 11:36:34

Unable to fetch data from Google Wifi

**Comment by RobyBS** [comment]
nessuno incontrando il problema , ha risolto in qualche modo ? 

sono ripartito da zero 4 volte .. ho persino usato la guida del sito ufficiale invece che indomus .. 

ma nulla da fare .... :( :( :( :( ...

**Comment by RobyBS** [comment]
scusate se rispolvero il thread ... 
ho provato ancora svariate volte .. e ho persino fatto altre 2 procedure ... ma nulla .. sempre stesso problema ... 

could not reach test , try later ...

non vorrei perdere ulteriori settimane e settimane, oltre che svariato tempo .. per un qualcosa che non si può più fare ...
grazie 1000 a chi mi risponderà ...

**Reply by essegi65** [reply]
Quale guida hai seguito?
Io ho integrato google home mini (anche io su hassio) seguendo [questa](https://indomus.it/guide/integrare-gratuitamente-google-home-assistant-con-home-assistant-via-gcp/). anche a me durante la configurazione su GCP mi ha reso un errore durante il test, ma dopo aver collegato il progetto a google assistant funzionano i comandi vocali impartiti a google su entità che si trovano su HA

**Reply by RobyBS** [reply]
ciao essegi65 .. grazie per avermi risposto ...

cosa intendi per "ma dopo aver collegato il progetto a google assistant" ?
io sono , diciamo , alle prime armi come avrai notato ... :( ...

ho seguito anche la guida che hai postato tu .. ma sempre ricevo lo stesso errore ... "could not reach test app , try later" ...

questa l'entità che voglio integrare come prova ... 

```
project_id: hassiohd-xxx
service_account: !include hassiohd-xxx.json
report_state: true
expose_by_default: false
entity_config:
  climate.a_c_soggiorno:
    name: A/C Soggiorno
    room: Soggiorno
```

ho fatto un file googlehome.yaml dedicato e inserito la stringa

`google_assistant: !include googlehome.yaml`  nel file configuration ...

**Reply by essegi65** [reply]
Segui fino in fondo la guida del link sopra e vedrai che una volta finito il progetto su GCP devi salvarti il tuo "prjectID" , andare su "service account key" e crearti il file con estensione json. sia il progectid che il file con estensione json sono da inserire all'interno di HA.
Fai prima a seguire passo passo la guida che farti spiegare i passaggi sul forum.
E non disperarti se qualche cosa va storto, riprova.

**Reply by RobyBS** [reply]
grazie 1000 per ora ! ... 
se a qualcuno funziona, dovrà funzionare anche a me prima o poi ... ;) ...
